plugins/woocommerce/client/legacy/js/admin/wc-shipping-zone-methods.js

```diff
@@ -197,16 +198,33 @@
 						model   = view.model,
 						methods   = _.indexBy( model.get( 'methods' ), 'instance_id' ),
 						changes = {},
-						instance_id = $( this ).closest('tr').data('id');
+						instance_id = $( this ).closest( 'tr' ).data( 'id' );
 
 					event.preventDefault();
-
-					delete methods[ instance_id ];
-					changes.methods = changes.methods || { methods : {} };
-					changes.methods[ instance_id ] = _.extend( changes.methods[ instance_id ] || {}, { deleted : 'deleted' } );
-					model.set( 'methods', methods );
-					model.logChanges( changes );
-					view.render();
+					shippingMethodView.block();
```

Comment:
We also discussed showing a confirmation dialog first, to see if the user really wanted to delete the shipping method, could we still add that?